docs/extensions/flex.md

```diff
@@ -0,0 +1,66 @@
+
+🤔 As many of you know, services like dial-a-ride are often brushed over by riders, who sometimes have no clue they even exist. This lack of accessibility is an issue for producers, consumers, and riders. Imagine a group of tourists arriving at your local airport and would like to reach a rural area that does not offer scheduled bus routes but only an on-demand bus service. The tourists check their preferred trip planner app and do not find a viable public transportation option; they end up renting a car. Being tourists, they missed all of your paper flyers posted along the hallway announcing the on-demand service. Not only is your service underutilized, but it lacks the discoverability to meet current and future rider demand. This is where GTFS-Flex comes in by providing that information to the rider allowing them to enjoy the services you worked hard to promote.
+
+<img src="../assets/flex-userjourney-resize.jpg" alt="GTFS-Flex User Journey">
+
+🔮 MobilityData expects GTFS-Flex to open the door to deeper standardization of demand responsive transportation including expansion into transactional and real-time components using GTFS-OnDemand. We are preparing a suggested strategy to best handle the growing number of modes of transportation and complexity of concepts engaging in this area.
+
+<a class="button no-icon" href="https://github.com/google/transit/issues/382" target="_blank">See the full proposal</a>
+
+## WIP: Base Implementation
+We have opened an issue to cover the base implementation of GTFS-Flex. Using this incremental approach should allow us to better establish a strong base for consensus.
+
+🔭 Scope of GTFS-Flex: Base Implementation
+
+Based on what has already been tested in the field, we see GTFS-Flex solving the following key use cases:
+
+- Display available demand responsive transportation services for rider discovery
+- Deviated bus routes
+- Dial-a-ride (e.g Rufbus, On-call bus)
+- Displaying contact information (phone number and/or website URL) for booking
+
+<a class="button no-icon" href=https://share.mobilitydata.org/slack>Join #gtfs-flex on Slack</a><a class="button no-icon" href=https://groups.google.com/g/gtfs-changes>Join GTFS-Changes Google Group</a>
+
+## Early Implementations
+
+- [MNDoT Flex Pilot Project: Trillium, IBI, Transit, MNDoT, Cambridge Systematics, and Token Transit](https://blog.transitapp.com/case-study/mndot-gtfs-flex-bringing-rural-riders-into-the-fold/) 
+- [Open Trip Planner](https://www.opentripplanner.org/)
+- [Mobility on Demand Sandbox in Vermont](https://www.connectingcommuters.org/)
+- [Tulare County Area Transit](https://ridetcat.org/)
+- [Northwest Oregon Transit Alliance (NW Connector)](https://www.nworegontransit.org/)
+- [Vamos Mobility App](https://vamosmobileapp.com/)
+- [RTD Denver Flexride](https://www.rtd-denver.com/services/flexride)
+
+Contact us to add your GTFS-Flex implementation to this page
```

Comment:
suggestion: This button should go to an email address rather than Slack, since the point of contact isn't clear if they're just directed to Slack.